src/TransactionalSession.MsSqlMicrosoftDataClient.AcceptanceTests/When_using_outbox_send_only.cs

```diff
@@ -0,0 +1,84 @@
+namespace NServiceBus.TransactionalSession.AcceptanceTests;
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyInjection;
+using AcceptanceTesting;
+using AcceptanceTesting.Customization;
+using Configuration.AdvancedExtensibility;
+using NUnit.Framework;
+
+public class When_using_outbox_send_only : NServiceBusAcceptanceTest
+{
+    [Test()]
+    public async Task Should_send_messages_on_transactional_session_commit()
+    {
+        //await OutboxHelpers.CreateOutboxTable<SendOnlyEndpoint>();
```

Comment:
@andreasohlund I dont this this line of code is needed. Thoughts?

Reply:
I would assume that you still need it but change it to `await OutboxHelpers.CreateOutboxTable<ProcessorEndpoint>();` since this will provision the table

Reply:
Try running the test with a clean database to make sure

Reply:
Yeah clarified it and it creates processor outbox tables